tests/test_completion.py

```diff
@@ -110,7 +121,7 @@ def test_completion_plugin(
     @pytest.mark.parametrize(  # type: ignore
         "prog", [["python", "hydra/test_utils/completion.py"]]
     )
-    @pytest.mark.parametrize("shell", ["bash"])  # type: ignore
+    @pytest.mark.parametrize("shell", get_supported_shells())  # type: ignore
```

Comment:
1. test both, but [skip](http://doc.pytest.org/en/latest/skipping.html) if fish is not installed.
2. install fish on in the [circleci config](https://github.com/facebookresearch/hydra/blob/master/.circleci/config.yml) on Linux and Mac jobs.

Reply:
I edited the circleci conf but I honestly have zero knowledge of mac. Let's see what happens.